.tekton/main-push.yaml

```
@@ -0,0 +1,54 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/app-sre/r-aws-msk?rev={{revision}}
```

It looks something happened with the name of the repo here

```suggestion
    build.appstudio.openshift.io/repo: https://github.com/app-sre/er-aws-msk?rev={{revision}}
```

And all over this file too 😅

---

You have 🦅 👀!